### PR TITLE
feat(ui): change sats symbol

### PR DESCRIPTION
**Pull request by pwltr**
### Description

₿

### Notes

Changes sats symbol to ₿ and adds a setting to switch to classic denomination.

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactoring (improving code without creating new functionality)

### Tests

- [x] Detox test
- [ ] Unit test
- [ ] No test

### Screenshot / Video

https://github.com/synonymdev/bitkit/assets/8538369/16ad8f78-fba1-49c6-99be-d3369174290f

### CODE REVIEW

**Review by Jasonvdb — APPROVED** [review]

File: src/store/utils/backup.ts [hunk 1]
```diff
@@ -374,6 +375,8 @@ const performSettingsRestore = async (): Promise<
 			updateSettings({
 				...expectedBackupShape,
 				...backup,
+				// @ts-ignore migrate unit
```
Comment (limpbrains) [inline src/store/utils/backup.ts]: you can check backupRes.value.metadata.version here and only apply this migration for old backups

Comment (pwltr) [reply src/store/utils/backup.ts]: Cool, could you push a change please?

Comment (limpbrains) [reply src/store/utils/backup.ts]: ok, I will

Comment (pwltr) [reply src/store/utils/backup.ts]: thanks

**Review by limpbrains — APPROVED** [review]

**Pull request merged**